runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/hotcold/HotColdTemperature.java

```diff
@@ -111,4 +111,19 @@ public static HotColdTemperature getFromTemperatureSet(final Set<HotColdTemperat
 			.max(Comparator.comparingInt(x -> (x.getText()).length()))
 			.orElse(null);
 	}
+
+	//This is used for dev mode only.
+	public static String getMessageFromDistance(int distance, Set<HotColdTemperature> temps)
```

Comment:
While this technically works as-is and is unlikely to break since it depends on hot-cold internals which are unlikely to change, it is completely dependent upon the ordering of `temps` and can be made to give incorrect results by passing a set with out-of-order temperatures. I'd like to see this made deterministic so that ordering would not matter.